### ISSUE TITLE
most don't work

### ISSUE THREAD

**Issue by yoint** [C4K3/minecraft-3-character-names#6]
Most names tagged "available now" don't actually work and they're already in use.

**Comment by C4K3** [comment]
Yeah it's not a live list, names get taken pretty quickly, and I don't update it very often.

Also there are some names that are blocked, but nobody has them so the Mojang API just reports them as available.

You pretty much have to run the script yourself right before you want a name if you want the best chances.

**Comment by yoint** [comment]
The thing is though, I do run the script right before I'm trying to get a name and even then 95% of supposedly available names are taken, and I'm not even talking about unique or offensive names. 

If for example I get these results, there's still a 95% chance they're all taken (based on my experience).

Setting 6ft to available-now
Setting 6g5 to available-now
Setting 6gz to available-now
Setting 6h7 to available-now

**Comment by C4K3** [comment]
That does look broken, 6ft is and has been taken for a long time. I'll look into why it's marking it as available.

**Comment by yoint** [comment]
Thank you for looking into it. I look forward to see if you find out what's wrong

**Comment by C4K3** [comment]
I think 6ft is a deleted account
```
https://api.mojang.com/users/profiles/minecraft/6ft
```
returns 204, while
```
https://api.mojang.com/user/profiles/70bbf2a2faef49db977b3bcad3c6f5f4/names
```
returns just that name "6ft". Can you confirm that it's not possible to change your name to 6ft?

Same thing with 6g5 and 6h7.

Namemc does show them as taken, but the Mojang API is not responding with any information on that name, which is why the script is marking them as available. I'm not sure which of those is correct, the only way to find out is to try to take the name.

(The script works the same as it used to, but either a bunch of accounts have been deleted or names blocked, or Mojang have changed their API.)

6gz was available but was taken on May 21st.

**Comment by yoint** [comment]
Thank you for your detailed response.
I took 4 random names in this case, I did not check to see if all of them were taken  I did in fact try to take 6ft on the Mojang website and it did not let me, prompting me the name was already in use. The same goes for a lot of names that are tagged as available. I'd try to change my name via Mojang but it wouldn't let me because they were in use

**Comment by C4K3** [comment]
Thanks for checking, sounds like the Mojang API has changed then. I'll try to see if there's another way to check if a name is available or not, and then change the script.

**Comment by yoint** [comment]
Hi C4K3,

Die you ever get around to test new ways to 
check if names are available?

**Comment by C4K3** [comment]
I tried looking into it, but I couldn't figure out a nice way of doing it without a reliable username -> UUID lookup.

Basically both the Playername -> UUID and Playernames -> UUIDs APIs are returning `no content` for some of the players (such as `6ft`.) If you look them up by UUID then it works, but if you look them up by playername then it doesn't work.

Unless there's some API endpoint I don't know of, or the current API gets changed back, this is really only solvable using some kind of Playername -> UUID database like namemc has. Maybe they should try implementing this feature. They already have a way of seeing 3 character names that will become available soon: https://namemc.com/minecraft-names?length_op=eq&length=3&lang=&searches=0

I would recommend just using that for now.
